src/components/Modal/Modal.tsx

```diff
@@ -31,7 +31,8 @@ const Header = styled.h1`
 `;
 
 const Content = styled.div`
-    overflow: auto;
+    overflow-y: auto;
+    overflow-wrap: break-word;
     padding: 1em 1.5em 1.5em;
     width: 100%;
     color: ${(props) => props.theme.main.foreground[1]};
```

Comment:
Won't let me comment below for some reason, is there a reason why CloseButton has border of 0. Also couldn't you just add the aria-label onto the CloseButton Opening Tag, instead of having to use `styled.button.attrs`

Reply:
Although border: 0 and border: none are equivalent, I personally prefer the latter. It's up to you though

Reply:
okie!